addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -126,9 +126,18 @@ class MessageList extends Component {
                 case 'change-of-thread-cache':
                     this._adjustFromChangeOfThreadCache(hint);
                     break;
+                case 'chat-window-unfolded':
+                    this._adjustFromChatWindowUnfolded(hint);
+                    break;
                 case 'current-partner-just-posted-message':
                     this._adjustFromCurrentPartnerJustPostedMessage(hint);
                     break;
+                case 'home-menu-hidden':
+                    this._adjustFromHomeMenuShown(hint);
+                    break;
+                case 'home-menu-shown':
+                    this._adjustFromHomeMenuHidden(hint);
+                    break;
```

Comment:
you mixed names (`hidden`/`Shown` & `shown`/`Hidden` => `hidden`/`Hidden` & `shown`/`Shown`)